### PR TITLE
added windows 10 supportt

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/9152
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/9200

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @e0i, @nam31, for your information

Thank you

File: windows/security/threat-protection/microsoft-defender-atp/network-protection.md [hunk 1]
```diff
@@ -49,9 +49,9 @@ You can also use [audit mode](audit-windows-defender.md) to evaluate how Network
 
 Network protection requires Windows 10 Pro or Enterprise, and Microsoft Defender Antivirus real-time protection.
 
-| Windows 10 version | Microsoft Defender Antivirus |
+| Windows version | Microsoft Defender Antivirus |
 |:---|:---|
-| Windows 10 version 1709 or later | [Microsoft Defender AV real-time protection](../microsoft-defender-antivirus/configure-real-time-protection-microsoft-defender-antivirus.md) and [cloud-delivered protection](../microsoft-defender-antivirus/enable-cloud-protection-microsoft-defender-antivirus.md) must be enabled |
+| Windows 10 version 1709 or later, Windows Server 1803 or later | [Microsoft Defender AV real-time protection](../microsoft-defender-antivirus/configure-real-time-protection-microsoft-defender-antivirus.md) and [cloud-delivered protection](../microsoft-defender-antivirus/enable-cloud-protection-microsoft-defender-antivirus.md) must be enabled |
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/network-protection.md:54]:
```suggestion
| Windows 10 version 1709 or later, Windows Server 1803 or later | [Microsoft Defender AV real-time protection](../microsoft-defender-antivirus/configure-real-time-protection-microsoft-defender-antivirus.md) and [cloud-delivered protection](../microsoft-defender-antivirus/enable-cloud-protection-microsoft-defender-antivirus.md) must be enabled |
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by illfated — APPROVED** [review]

**Comment by illfated** [comment]
Ref. the PR title: it would be more correct to say **added Windows Server support** (just in case you didn't notice).

**Review by denisebmsft — APPROVED** [review]
Thank you @MaratMussabekov @JohanFreelancer9 @illfated

**Comment by illfated** [comment]
> Ref. the PR title: it would be more correct to say **added Windows Server support** (just in case you didn't notice).

@MaratMussabekov : Would it be OK if I ask you to edit the PR title as suggested above? Thanks in advance.

**Comment by illfated** [comment]
Oh well, never mind. Not a big deal. Fair enough.

**Pull request merged**